### DISCUSSION TITLE
Where is text-primary?

### DISCUSSION THREAD

**Discussion by sajt** [mdbootstrap/TW-Elements#1836] [Support from community] [answered]
I am working on a Laravel project, and I like to use Tailwind-Elements. In the examples there are a lot of text-primary-700 text-secondary-700 and so on. But I do not find the definition these selectors. Where they are?

**Comment by juujisai** [comment, marked as answer]
They come from the Tailwind Elements plugin.cjs file:

https://github.com/mdbootstrap/Tailwind-Elements/blob/master/src/js/plugin.cjs
